### DISCUSSION TITLE
Android Auto navigation dismissed when leaving the car.

### DISCUSSION THREAD

**Discussion by styrheim** [osmandapp/OsmAnd#15566] [General]
If I bring the phone with me, when I stop at a café halfway in a long trip, the current navigation is automatically dismissed.

That is not what I need, I want to continue the current voyage, when I have finished the coffee.

(I think the current navigation continues, if I turn off the ignition, but leave the phone plugged to the car?)

Android Auto 8.2.623924-release
Osmand+ 4.2.7

**Comment by vshcherb** [comment]
It's designed by Android Auto policy, they even check this behavior during manual review. If I'm not mistaken we can't influence that or we need to do some special ui for that.

**Reply by styrheim** [reply]
I tested with Google Maps. Like Osmand, the navigation was dismissed when I unplugged the phone.

But – unlike Osmand – the former navigation reappeared when I plugged the phone back into the car.

So there seems to be a way to get what I want withing the Android Auto policies?

**Reply by vshcherb** [reply]
hm.. that's possibly what needs to be implemented on our side.

**Comment by GingerZero** [comment]
I have run into this same issue while going on longer road trips and sometimes even just by filling up the car on shorter trips.

My most frustrating experience was when visiting family ~1400km (~875mi). There were many stops and having to go in and make the route recalculate every time was extremely time consuming as it could take around 5min to actually make the route.
Before the trip back I figured out that the easiest way to not have to sit idle while the app calculates was to calculate the route in advance and save it as a track. Plan route > wait for calculation > once done, hit details below the elevation graph > on the top right select the save icon and save the track. With this at least, OsmAnd could resume navigation from my current position rather than need to recalculate everything.

Only disadvantage is that I don't know how navigation following a track would behave if an unforseen event makes you take a deviation... Will OsmAnd recalculate for the destination or try to get back on track as quickly as possible...

Anyway... I thought I'd mention it as I believe it could be a possible way to make the navigation be "resume-able" when making fuel stops and decoupling the phone from the car in the middle of navigation.

PS: I'll try to have a look whether it is possible to save the navigation state somehow and check for this navigation state when connecting to android auto. But seeing as I am no developer/programmer (certainly not in java/kotlin) I can't make any promises on whether I can/will make it :(

**Reply by DmitryAlexei** [reply]
you can leave your comment here https://github.com/osmandapp/OsmAnd/issues/18122
